releaser/make_release.py

```diff
@@ -132,41 +131,6 @@ def create_source_archive(build_dir, package_name, release_name, rev, **extra_kw
     echocall(['git', 'archive', '--format', 'zip', '--output', archive_name, rev])
 
 
-def update_version(build_dir, release_name, package_name, module_name, public_release, **extra_kwargs):
```

Comment:
Why did you move update_version and set_config to utils? I see utils as some generic stuff which could be used in other contexts while both update_version and set_config are quite specific to the releaser. Besides update_version is a "step" function and should IMO be kept with other step functions. set_config is "almost" a step function (it is a kind of mandatory first step) so I believe it belongs more with the other steps than in utils.py but I admit this is subjective and debatable.

Reply:
I asked you to review this PR because I wasn't sure about this move.

An error was raised when I called next_release.py for the first time because the argument `public_release` wasn't actually passed to the step-function `update_version` (defined originally in make_release.py). 
`public_release` is defined in `step_config`. 

IMO (but I'm open for debate), we should use a `config` dict in all modules (except utils.py of course). 
The idea is to have some homogeneity among step-functions (they should all rely on the same mechanism -> define a global `config` dict) wherever they are defined.  

I thought that `utils.py` was a place to put "common" stuff and that's why I moved `set_config` in it.
The step-function `update_version` is used in both make_release.py and next_release.py and I wanted to put it in "common" place.
I didn't realized that functions defined in `utils.py` could be used in other contexts than the releaser (If I understand you correctly). If that so, I'm OK to move back `set_config` and `update_version` to make_release.py while they are not only used in make_release.py. I don't think adding an extra `common.py` module is really necessary.

Reply:
> If you mean a dict named config passed as parameters for step functions

Well, I wasn't clear, my bad. I don't want to pass the dict as it to step-functions but 

1. to call step-functions as `func(**config)`,
2. to have one common `config` dict for all modules

step-functions will still be declared with explicit arguments and a last **extra_kwargs argument.